openmc/mgxs/mgxs.py

```diff
@@ -15,51 +14,67 @@
 
 
 # Supported cross section types
-MGXS_TYPES = ['total',
-              'transport',
-              'nu-transport',
-              'absorption',
-              'capture',
-              'fission',
-              'nu-fission',
-              'kappa-fission',
-              'scatter',
-              'nu-scatter',
-              'scatter matrix',
-              'nu-scatter matrix',
-              'multiplicity matrix',
-              'nu-fission matrix',
-              'scatter probability matrix',
-              'consistent scatter matrix',
-              'consistent nu-scatter matrix',
-              'chi',
-              'chi-prompt',
-              'inverse-velocity',
-              'prompt-nu-fission',
-              'prompt-nu-fission matrix']
+MGXS_TYPES = [
```

Comment:
Some of these global types are mutable but never modified in our code. Would it be better for them to be tuples?

Reply:
Indeed, I'll convert them to tuples.